site/src/components/form.md

```diff
@@ -110,8 +110,11 @@ If you want to skip the basic validation on an input, apply a `data-skip-validat
 		loc-input-max-length="{inputLabel} cannot be longer than {maxLength} characters."
 		loc-input-min-length="{inputLabel} must be at least {minLength} characters."
 		loc-input-required="{inputLabel} is required."
+		loc-not-authenicated="You are not authenticated, please refresh the page and try again. If this issue persists please log out and log back in."
```

Comment:
This should be `loc-not-authenticated` (it's missing a `t`), and these sentences should be split.

```suggestion
		loc-not-authenticated="You are not authenticated. Please refresh the page and try again. If this issue persists, please log out and log back in."
```